integration-tests/chopsticks/src/network/hydraDx.ts

```diff
@@ -0,0 +1,69 @@
+import { setupContext, SetupOption } from '@acala-network/chopsticks-testing'
+import type { Config } from './types.js'
+import * as SpiritnetConfig from './spiritnet.js'
+import { initialBalanceHDX, initialBalanceKILT, toNumber } from '../utils.js'
+
+export const options: SetupOption = {
+	endpoint: process.env.HYDRADX_WS || ['wss://hydradx-rpc.dwellir.com', 'wss://rpc.hydradx.cloud'],
+	db: './db/hydradx.db.sqlite',
+	port: toNumber(process.env.HYDRADX_PORT) || 9001,
+}
+
+export const kiltTokenId = 60
+
+export function setGovernance(addr: string[]) {
+	return {
+		TechnicalCommittee: { Members: addr },
+		Council: { Members: addr },
+	}
+}
+
+export function assignNativeTokensToAccount(addr: string[], balance: bigint = initialBalanceHDX) {
```

Comment:
`
```suggestion
export function assignNativeTokensToAccounts(addr: string[], balance: bigint = initialBalanceHDX) {
```

Reply:
updated in: [41c2859](https://github.com/KILTprotocol/kilt-node/pull/614/commits/41c2859fbd6ce083c9dc7fc8ea53a50bc36fb37a)